File: 2024_01_January/26_Friday/palindrome_int.go
func isPalindrome(x int) bool {
    str := strconv.Itoa(x)
    length := len(str)
    
    for i := 0; i < length / 2; i++ {
        if str[i] != str[length - i - 1] {
            return false
        }
    }

    return true
}

File: 2024_01_January/26_Friday/palindrome_int.cpp
class Solution
{
public:
  bool isPalindrome(int x)
  {
    string str = to_string(x);
    int len = str.length();

    for (int i = 0; i < len / 2; i++)
    {
      if (str[i] != str[len - i - 1])
      {
        return false;
      }
    }

    return true;
  }
};